### ISSUE TITLE
[Backend] As a user, I can upload a CSV file with keywords

### ISSUE THREAD

**Issue by Tuubz** [nimblehq/ic-rails-huey-toby#12]
## Why

A table row must be created for every keyword in the CSV file.

## Acceptance Criteria

- Create a new table for keywords, besides the default fields, add a name field.
- Add references with the users table (one user has many keywords & one keyword has one user)
- Create a new endpoint: `POST ap/v1/upload?engine={google/bing}`
    - **Request**: CSV file
    - **Response**: Success message
- Start one background job that processes the keywords into the table.

**Note**: Make use of `sidekiq` to handle the background job

## Design
N/A

## Resources
- https://www.postman.com/postman/workspace/postman-answers/documentation/13455110-00378d5c-5b08-4813-98da-bc47a2e6021d
- https://github.com/sidekiq/sidekiq

**Comment by olivierobert** [comment]
@Tuubz I recommend leaving out the background job setup from this story. The CSV upload and scraping can be built in parallel (by separate devs). Then there can be another story to integrate the two features together. I also predict that you will face some challenges in testing the background job as it is a bit different than other type of testing. So having a separate task would make it easier to estimate.

**Comment by Tuubz** [comment]
@olivierobert Noted, thank you. I'll separate it 🙏
